bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -45,6 +45,7 @@ using std::chrono::system_clock;
 using namespace std::placeholders;
 using namespace concord::diagnostics;
 using namespace concord::util;
+using bftEngine::impl::Digest;
```

Comment:
Once you modify namespace, I don't think this is required.

Reply:
Done.